salt/client/mixins.py

```diff
@@ -516,7 +516,7 @@ def _gen_async_pub(self, jid=None):
         tag = salt.utils.event.tagify(jid, prefix=self.tag_prefix)
         return {'tag': tag, 'jid': jid}
 
-    def async(self, fun, low, user='UNKNOWN', pub=None):
+    def asynchronous(self, fun, low, user='UNKNOWN', pub=None):
```

Comment:
I was concerned about this one, but it looks like this is an internal api, what should be used is `cmd_async` which calls `self.async` or now `self.asynchronous`

Reply:
This should be fine, and not break anything.